### ISSUE TITLE
Results for training on DTU

### ISSUE THREAD

**Issue by fatbao55** [bennyguo/instant-nsr-pl#111]
Thanks for the great work! 

I would like to check what is your PSNR for training on DTU. Mine  is 3.137 and  6.637 without and with mask respectively. It seems rather low compared to the values reported when trained on Nerf-Synthetic (30+).

Is this abnormal?

**Comment by AIBluefisher** [comment]
It is absolutely abnormal. The PSNR on DTU should all be larger than 20.

**Comment by anonymous-pusher** [comment]
You might have skipped the part in the readme saying that :

> PSNR in the testing stage is meaningless, as we simply compare to pure white images in testing.


The current dataloader for dtu does not load ground truth images, but only uses white pixels. You will need to define your own train/test splits in the dataloader or set it to use the training views during testing.
you can check the line https://github.com/bennyguo/instant-nsr-pl/blob/e5fe3f246cf2d512494a73c727174c3ca1c3c695/datasets/dtu.py#L124

**Comment by fatbao55** [comment]
@mJones00 You're right, thanks for the clarification!
